plugins/src/webrtcsink/imp.rs

```diff
@@ -1970,39 +1578,28 @@ impl WebRTCSink {
         Ok(())
     }
 
-    fn process_loss_stats(
-        &self,
-        element: &super::WebRTCSink,
-        peer_id: &str,
-        stats: &gst::Structure,
-    ) {
+    fn set_bitrate(&self, element: &super::WebRTCSink, peer_id: &str, bitrate: u32) {
         let mut state = element.imp().state.lock().unwrap();
-        if let Some(mut consumer) = state.consumers.get_mut(peer_id) {
-            if let Some(congestion_controller) = consumer.congestion_controller.as_mut() {
-                congestion_controller.loss_control(&element, stats, &mut consumer.encoders);
-            }
-            consumer.stats = stats.to_owned();
-        }
-    }
 
-    fn process_stats(&self, element: &super::WebRTCSink, webrtcbin: gst::Element, peer_id: &str) {
-        let peer_id = peer_id.to_string();
-        let promise = gst::Promise::with_change_func(
-            glib::clone!(@strong peer_id, @weak element => move |reply| {
-                if let Ok(Some(stats)) = reply {
-
-                    let mut state = element.imp().state.lock().unwrap();
-                    if let Some(mut consumer) = state.consumers.get_mut(&peer_id) {
-                        if let Some(congestion_controller) = consumer.congestion_controller.as_mut() {
-                            congestion_controller.delay_control(&element, stats, &mut consumer.encoders,);
-                        }
-                        consumer.stats = stats.to_owned();
-                    }
+        if let Some(consumer) = state.consumers.get_mut(peer_id) {
+            let fec_ratio = {
+                if bitrate <= 2_000_000 || consumer.cc_info.max_bitrate <= 2_000_000 {
```

Comment:
How is that related to 2MBit/s? Maybe worth a comment :)

Reply:
Not that this is just moved existing code, thibault isn't the guilty party here :)